### ISSUE TITLE
Everyone's name and email

### ISSUE THREAD

**Issue by minin43** [akstoulil/Group-Project-1#15]
I'm the guy who suggested we set this up. reply with your name, your email, and which one of the two austins you are

**Comment by akstoulil** [comment]
Austin Stoulil
akstoulil@dmacc.edu

Hope this helps!

**Comment by minin43** [comment]
Logan Christianson
lchristianson1@dmacc.edu

**Comment by minin43** [comment]
ALSO, a thought about the powerpoint and project report:

The project report is split into 4 separate categories (as seen on BlackBoard): Introduction, Goals, Challenges & Obstacles, and Accomplishments. This could be a good structure to follow for the powerpoint. Each person is assigned to talk about one of those specific topics regarding our project.

**Comment by akstoulil** [comment]
Hello everyone! Just checking in on how things for the powerpoint and word document are going. Is everything going along smoothly, and does anyone need any help with anything? Any feedback is appreciated!

**Comment by minin43** [comment]
Still waiting on getting everyone's email so I can share the doc with everyone. Austin if you could write a bit under the section I have marked with bold letters, that would be appreciated.

**Comment by akstoulil** [comment]
Sounds good. I think that the the file you sent me on google drive/google docs needs permission to be edited. I just sent out a request to have permission granted. It should be sent to your gmail account.

**Comment by minin43** [comment]
Alright, sent out the new invitation.

**Comment by akstoulil** [comment]
I think it should be sent out, is it not in your mail quite yet?

**Comment by CrazyCarl78** [comment]
Craig Carlson
cacarlson8@dmacc.edu.

**Comment by akstoulil** [comment]
Hi everyone, just checking in on how things are going! I made a few edits to the word document, so if everyone could look it over and give some feedback on it that would be good. 

Link to the word document: https://docs.google.com/document/d/1uBKDd3MmTWBnshd14UH9CO-vzlCsi6w3z9NntOAsB1g/edit?ts=5a9b0a76

**Comment by agkingrey** [comment]
Hey there everyone, Here's a good majority of the slideshow. I like the idea of splitting up the sections. On top of that we should each talk about the sections of code we worked on, that I have screen capped, and labeled in the slideshow. Also, I am open to any suggestions of other content we can put in here. I tried to remember all the stuff we talked about, and added that to the slideshow. 
My email is: mr.kingrey04@gmail.com

**Comment by agkingrey** [comment]
https://docs.google.com/presentation/d/1ajRwHq32HfpSEKGn22h9sOgxLwiVCg6HqKcJsgeNpB0/edit?usp=sharing

**Comment by akstoulil** [comment]
Hi Austin, thanks for getting back to us on how the power point is going! I think it looks pretty good so far. We will probably each want to make sure that we look over the slides, and decide what we should each talk about in regards to what we each did. Thanks for working on it!

**Comment by CrazyCarl78** [comment]
Great job on the power point!  The report looks good too! Thank you for working on it!

**Comment by akstoulil** [comment]
Hello everyone! Just checking in again on things. I was just wondering what everybody thinks of the status of our project, and of our word document and power point. If nobody has any objections or suggestions for the power point or word document, I could try to get it turned in for us tonight. This way we can have everything turned in and ready to go for class a little bit before the due date. Any feedback on this is appreciated!

**Comment by minin43** [comment]
No serious objections to the powerpoint, but how are we looking to split it up? What are the sections to talk about?

**Comment by akstoulil** [comment]
We are probably going to want to split up the power point by who worked on what. I know that in the slides there is a section for the FunctionMain file, so in your case you could probably talk about that and explain how setting up the deck of cards works. I figure that everyone will probably just go into a little detail about what we each did on the project and such.

**Comment by CrazyCarl78** [comment]
I don't have any objections.

**Comment by akstoulil** [comment]
Hello everyone. Just giving an update on turning in the project. I'll try to turn it in here tonight soon, so if you have any suggestions for the word document or power point now is the time to post them. Any feedback is appreciated!

**Comment by akstoulil** [comment]
Hello again everyone! Just thought I should let you all know that I just got it turned on Blackboard. Now all that is left to do is just practice what each of us will say for the presentation on Thursday.

**Comment by minin43** [comment]
The presentation is going to be an absolute mess if we don't know who's going to say what going into it.

**Comment by agkingrey** [comment]
I went ahead and assigned all the slides, feel free to add more info to your section, or do what you need to get prepared. Check the bottom left corner of each slide.  Everyone got a non-coding slide, and then their section they worked on per the word doc we submitted. See you guys tomorrow.
